### ISSUE TITLE
Allow phi 1.5 to accept attention mask as a parameter

### ISSUE THREAD

**Issue by bnicholl** [huggingface/transformers#26397]
### Feature request

As of right not, phi-1.5 does not accept attention masking for training or running inferences in batches. Could we get phi-1.5 to accept attention mask as a parameter

### Motivation

I would like attention masking for fine tuning

### Your contribution

If this isn't planned on getting taken care of, I could begin incorporating the weights into a torch module that will accept attention mask myself.

**Comment by LysandreJik** [comment]
Hey @bnicholl, phi 1.5 is being added to the library here: https://github.com/huggingface/transformers/pull/26170

This will support the attention mask as a parameter.
